### ISSUE TITLE
Important Read Respondus LockDown can see screen recording 

### ISSUE THREAD

**Issue by fri12013** [Totsukawaii/UndownUnlock#14]
hi guys just whant to let you knew so that you don't caught red-handed

if whant to do this bypass  Your organization can enable screen recording and see you going into chatgt or what ever and you will be caught red-handed

do this if you are sure that they don,t record your screen or on face to face tests

  
here you can see
https://support.respondus.com/hc/en-us/articles/17641623561371-How-does-the-screen-recording-option-in-Respondus-Monitor-work

**Comment by yoyomomo21** [comment]
is there a specific way to see if they are recording your screen? i think it says 'recording' at the top right for sure for the webcam, im not sure if thats also the case for screen recording

**Comment by fri12013** [comment]
yes it says recording but dosnot say what is being recorded idon,t knew if If they ware screen recording writeing something else then recording

**Comment by fri12013** [comment]
In the past, they used to write the thing that was being recorded, for example, they used to write it the top right (webcam recording)

**Comment by yoyomomo21** [comment]
I guess it's safe to assume, if it says recording while the webcam is not enabled then it's screen recording, but if it's recording with webcam then assume the worst

**Comment by Totsukawaii** [comment]
I doubt lockdown browser can record your desktop screen. I'm guessing the most it is doing is only recording the browser window in Lockdown Browser itself so even if you alt tab I don't think it will be able to see it. This is based on what I've gathered from trying to bypass the DRM feature which prevents screen capture of lockdown browser, though I can't prove this.

If you are interested in what LDB is actually sending you can use https://github.com/mitmproxy/mitmproxy to man in the middle the connection to the respondus server the recordings are sent to. the Domain is monitor.respondus... something. In my case, I can confirm only Webcam recordings are being sent to the server

**Comment by yoyomomo21** [comment]
iv emailed Respondus Support and here is what they said:
```
Please see https://support.respondus.com/hc/en-us/articles/17641623561371-How-does-the-screen-recording-option-in-Respondus-Monitor-work
 
Students using Macs or Chromebooks will need to be sure that the screen recording option is enabled for LockDown Browser, but there is no setting to specifically alert them when screen recording is enabled for a particular test.
```

then i asked "Alright so an any case, if the student does bypass it somehow, I would be able to see the whole screen and the way it was bypassed, correct?"

reply:

```
If the screen is being recorded, the instructor would be able to see the whole screen.
```

**Comment by Totsukawaii** [comment]
> iv emailed Respondus Support and here is what they said:
> 
> ```
> Please see https://support.respondus.com/hc/en-us/articles/17641623561371-How-does-the-screen-recording-option-in-Respondus-Monitor-work
>  
> Students using Macs or Chromebooks will need to be sure that the screen recording option is enabled for LockDown Browser, but there is no setting to specifically alert them when screen recording is enabled for a particular test.
> ```
> 
> then i asked "Alright so an any case, if the student does bypass it somehow, I would be able to see the whole screen and the way it was bypassed, correct?"
> 
> reply:
> 
> ```
> If the screen is being recorded, the instructor would be able to see the whole screen.
> ```

Then again I doubt respondus support actually knows how the software works. You can see the cursor in the image in [the article](https://support.respondus.com/hc/en-us/articles/17641623561371-How-does-the-screen-recording-option-in-Respondus-Monitor-work) doesn't actually appear like how the cursor actually looks like. So it seems be be a virtual render of the cursor, which would align with my theory LDB doesn't actually capture the screen, and only captures the DRM protected content in LDB itself.

**Comment by fri12013** [comment]
Yes, i see that but you must bee Careful if you are copy and paste they will see that For Sure

**Comment by fri12013** [comment]
go to 50:30 here you can see it in video

https://www.youtube.com/watch?v=_i6jBzY8fFs

**Comment by fri12013** [comment]
I assume that if the organization does not instruct Mac users to give LDB permissions to screen record then they are not doing so because Mac owners must allow LDB to record their screen.
